ldi-orchestrator/ldio-common/pom.xml

```diff
@@ -25,6 +25,23 @@
             <artifactId>spring-boot-configuration-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+            <artifactId>request-executor</artifactId>
```

Comment:
This seems wrong.. Ldi common should be common. If it contains specific ldes client code, the integrity of the common module is compromised